src/lib.rs

```diff
@@ -664,6 +664,24 @@ impl<'a, T: Clone + Num + Neg<Output = T>> Neg for &'a Complex<T> {
     }
 }
 
+impl<T: Clone + Num + Inv<Output = T>> Inv for Complex<T> {
```

Comment:
I don't think we need `T: Inv`, but we do need `T: Neg` for the inherent `fn inv`.